tasks/install_shell.sh

```diff
@@ -681,6 +681,11 @@ case $platform in
     ;;
 esac
 
+if [[ -n "$PT_absolute_source" ]]; then
+  download_url=$PT_absolute_source
+fi
+
+download_filename="${tmp_dir}/${filename}"
```

Comment:
Why is this set here? It looks like it is handled in the `$PT__noop == true` case.

Reply:
Hi @donoghuc, thanks for reviewing.

You're absolutely right!
I was revisiting some old PR's; and I'm ashamed to admit that I haven't reviewed the updates to this file in the last two years...

Anyway; this is fixed in the latest commit.

Reply:
Awesome! thanks. Going to let CI run against this, will try to get it merged this afternoon. Appreciate the contribution!